packages/gestalt/src/SideNavigationTopItem.js

```diff
@@ -25,8 +26,8 @@ import Text from './Text';
 import { type Indexable } from './zIndex';
 
 export const NESTING_MARGIN_START_MAP = {
-  '0': 'var(--space-400)',
-  '1': 'var(--space-1200)',
+  '0': TOKEN_SPACE_400,
```

Comment:
will this have `px` added? Or does it need that?

Reply:
TOKEN_SPACE_400 is equivalent to 'var(--space-400)'

Yes, it will refer to the value with px.